bindings/matrix-sdk-ffi/src/api.udl

```diff
@@ -7,8 +7,23 @@ interface ClientError {
 
 callback interface ClientDelegate {
     void did_receive_auth_error(boolean is_soft_logout);
+    void did_receive_notification(NotificationItem notification);
```

Comment:
Why is this part of the same delegate?

Reply:
shoul it be a different one?
It just made the implementation a bit easier to use the same existing one

Reply:
To me it makes more sense to have different ones, so they're basically just replacements for closures. A client doesn't necessarily always want to receive both.